src/ui/components/shared/SharingModal/PrivacyDropdown.tsx

```diff
@@ -46,6 +46,30 @@ function DropdownButton({ disabled, children }: { disabled?: boolean; children:
   );
 }
 
+function filterByWorkspaceType(recording: Recording, workspace: Workspace) {
+  if (workspace.isTest && recording.metadata?.test) {
+    return true;
+  } else if (!workspace.isTest && !recording.metadata?.test) {
+    return true;
+  }
+
+  return false;
+}
+
+function sortWorkspaces(recording: Recording, a: Workspace, b: Workspace) {
+  if (a.id === recording.workspace?.id) {
+    return -1;
```

Comment:
This seems odd to me. I think the intention is to always put the workspace a recording is currently assigned to at the top of the list?

I think we should leave the list alpha sorted and allow the browser select control to do its thing. Mixing alpha-sorting like this _feels_ like a weird UX decision.

Reply:
That's fine. I think the sorting is only a problem for us with many teams anyway.